bins_h: 30
bins_v: 20
nb_obstacles: 200
min_range: 0.1  # [m]
max_range: 5  # [m]
hfov: 90  # [deg]
vfov: 106  # [deg]
body_frame: imu
# input_topic: /tof
# output_topic: /mavros/tof_obstacles
